### PR TITLE
Add list of supported languages

### PR DESCRIPTION
**Pull request by lzanita09**
Also realized that the localization documentation is within the tracking section, moved it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -365,11 +371,29 @@ let controller = ConnectButtonController(connectButton: self.connectButton,
...
 If no translations are found for a locale passed in for this parameter, the Connect Button and the corresponding flow will be displayed in English. If no value is provided for the `locale` parameter, a default value of `Locale.current` will be used.
 
-#### Anonymous ID
-By default, the SDK will track user interactions when users interact with the ConnectButton. In order to distinguish unique installs, we randomly generate a UUID per application installation (“anonymous id”), and send it along with the event requests.
-
-#### Disable tracking
-You may set `ConnectButtonController.analyticsEnabled = false` if you wish to opt-out from tracking. After this method is called, all tracking will be disabled for all of the ConnectButton instances within the app for as long as it is in-memory. If you want to persist the user's preference for disabling tracking, you should store the preference within your persistent storage, and set this variable every time the app is started.
+Text translation is supported for the following languages:
+* English (Default)
+* English - United Kingdom (en-rGB)
```
Comment (ssathy2) [inline README.md:376]: We should use the iOS locale identifier for these.

Comment (lzanita09) [reply README.md:376]: Do we have those somewhere?

Comment (lzanita09) [reply README.md]: Updated: https://github.com/IFTTT/ConnectSDK-iOS/pull/233/commits/ab851aa72aba62d96b33ecfacbcad821d51d62d4.

Is the locale identifier the value that we attach to the URL?

Comment (ssathy2) [reply README.md]: Yep, that's correct.

**Pull request merged**